src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java

```diff
@@ -42,34 +54,12 @@ public boolean getBulletExist() {
         return this.bulletExist;
     }
 
-    /**
-     * Description: The function returns the type of the shooter.
-     *
-     * @return a String with the type of the shooter.
-     */
-    public String getType() {
-        return this.type;
-    }
-
-    /**
-     * Description: The function returns the type of the shooter.
-     *
-     * @return a String with the type of the shooter.
-     */
-    public void goForward() {
-        if (this.getType() == Space.ALIEN) {
-            goDown();
-        } else {
-            goUp();
-        }
-    }
-
     /**
      * Description: The function displays a bullet shoot.
      * from the Spaceship.
      *
      */
-    public void isShootingToAlien(final Spaceship spaceship) {
+    public void isShootingToAlien() {
```

Comment:
Rename the metod of isShootingToSpaceship

Reply:
The method is to say who "is shooting to Alien"? A Spaceship